test/javascript/public/test/form_builders/validateSimpleForm.js

```diff
@@ -12,15 +12,16 @@ QUnit.module('Validate SimpleForm', {
     dataCsv = {
       html_settings: {
         type: 'SimpleForm::FormBuilder',
-        error_class: 'error small',
```

Comment:
`small` presence had a reason:

https://github.com/DavyJonesLocker/client_side_validations-simple_form/commit/399f3891b494c7073faad19f76cf79ab849fcbbe

Unfortunately this commit didn't come with a test, so we are not explicitly testing this behavior. 🙏🏼

Reply:
Don't worry, the problem is the missing test, or at least a comment should be present

Reply:
Implemented an explicit test for this use case in CSV: https://github.com/DavyJonesLocker/client_side_validations/commit/a49dd74a14b259b9a08551fedd163cf2406ba25c

Will do the same here